networking/eth2/src/test-support/java/tech/pegasys/artemis/networking/eth2/Eth2NetworkFactory.java

```diff
@@ -29,140 +27,123 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.hyperledger.besu.metrics.noop.NoOpMetricsSystem;
-import tech.pegasys.artemis.networking.eth2.peers.Eth2PeerManager;
-import tech.pegasys.artemis.networking.p2p.libp2p.LibP2PNetwork;
 import tech.pegasys.artemis.networking.p2p.network.NetworkConfig;
 import tech.pegasys.artemis.networking.p2p.network.P2PNetwork;
-import tech.pegasys.artemis.networking.p2p.network.PeerHandler;
-import tech.pegasys.artemis.networking.p2p.network.Protocol;
 import tech.pegasys.artemis.storage.ChainStorageClient;
-import tech.pegasys.artemis.storage.HistoricalChainData;
 import tech.pegasys.artemis.util.Waiter;
 
 public class Eth2NetworkFactory {
 
   protected static final Logger LOG = LogManager.getLogger();
   protected static final NoOpMetricsSystem METRICS_SYSTEM = new NoOpMetricsSystem();
-  private static final int MIN_PORT = 6000;
-  private static final int MAX_PORT = 9000;
+  protected static final int MIN_PORT = 6000;
+  protected static final int MAX_PORT = 9000;
+  protected List<Eth2Network> multiaddrpeers = new ArrayList<>();
 
-  private final List<Eth2Network> networks = new ArrayList<>();
+  protected final List<Eth2Network> networks = new ArrayList<>();
 
-  public Eth2P2PNetworkBuilder builder() {
```

Comment:
I'd added a `create` method to the `Eth2NetworkBuilder` in order to allow creating new instances of the builder when needed. I removed the builder because (iirc) I found the code in this builder to be duplicated.

Reply:
I found the error that my change introduced, so I've reverted the builder with the hopes to clean up duplicated code in the future.